CHANGELOG.asciidoc

```diff
@@ -3,6 +3,41 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.7.1]]
+=== Beats version 8.7.1
+https://github.com/elastic/beats/compare/v8.7.0\...v8.7.1[View commits]
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Fix Beats started by agent do not respect the allow_older_versions: true configuration flag {issue}34227[34227] {pull}34964[34964]
+- Fix performance issues when we have a lot of inputs starting and stopping by allowing to disable global processors under fleet. {issue}35000[35000] {pull}35031[35031]
```

Comment:
```suggestion
- Beats started by agent now configure one global instance of each default add_x_metadata processor instead of creating one processor per input. Fixes a significant performance degradation for the aws-s3 input. {issue}35000[35000] {pull}35031[35031]
```

This changelog entry isn't quite right, the primary fix was creating the default processors as Beat global processors.